flow-template/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,16 @@
+[[extensions]]
+name = "{{ name }}"
+type = "flow_template"
+handle = "{{ handle }}"
+
+title = "My Awesome Template"
+
+description = "A Template that helps increase merchant productivity"
+
+categories = ["orders", "fraud"]
+
+require_app = false
+
+visible = true
+
+enabled = true
```

Comment:
Check in #unified-extension-config about the fields, to be sure that it makes sense with the new unified effort.
For instance: having `name` and `title` is confusing 🤔